code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1001,7 +1001,7 @@
 		//(Using your gloves' nanochips, you/You) ( /quickly/expertly) start to lift Grey Tider onto your back(, while assisted by the nanochips in your gloves../...)
 		if(do_after(src, carrydelay, TRUE, target))
 			//Second check to make sure they're still valid to be carried
-			if(can_be_firemanned(target) && !incapacitated(FALSE, TRUE))
+			if(can_be_firemanned(target) && !incapacitated(FALSE, TRUE) && !target.buckled)

```

Comment:
Maybe add  `&& !target.buckled` to `can_be_firemanned()`?

Reply:
That makes it so that you can't even attempt to lift a buckled individual onto yourself, as it won't call `fireman_carry()` at all when doing `MouseDrop_T()`.

I personally really like allowing the players to attempt to lift someone that is buckled with having the progress bar fill just to get a generic failure line and have to realize that they did something kind of dumb, so I'm really happy with how it works without adding that check to `can_be_firemanned()`